modules/imaging_browser/jsx/imagingBrowserIndex.js

```diff
@@ -0,0 +1,199 @@
+         * queried columns in _setupVariables() in imaging_browser.class.inc
+         */
+        const options = this.state.data.fieldOptions;
+        const configLabels = options.configLabels;
+        const fields = [
+            {label: 'Site', show: true, filter: {
+                name: 'site',
+                type: 'select',
+                options: options.sites,
+            }},
+            {label: 'PSCID', show: true, filter: {
+                name: 'PSCID',
+                type: 'text',
+            }},
+            {label: 'DCCID', show: true, filter: {
+                    name: 'DCCID',
+                    type: 'text',
+            }},
+            {label: 'Project', show: true, filter: {
+                name: 'project',
+                type: 'select',
+                options: options.projects,
+            }},
+            {label: 'Vist Label', show: true, filter: {
+                    name: 'visitLabel',
+                    type: 'text',
+            }},
+            {label: 'Visit QC Status', show: true, filter: {
+                    name: 'visitQCStatus',
+                    type: 'select',
+                    options: options.visitQCStatus,
+            }},
+            {label: 'First Acquisition', show: true},
+            {label: 'First Insertion', show: true},
+            {label: 'Last QC', show: true},
+            {label: 'New Data', show: true},
+            {label: 'Links', show: true},
+            {label: 'SessionID', show: false},
+            {label: 'Sequence Type', show: false, filter: {
+                name: 'sequenceType',
+                type: 'select',
+                options: options.sequenceTypes,
+            }},
+            {label: 'Pending New', show: false, filter: {
+                name: 'pendingNew',
+                type: 'select',
+                options: options.pendingNew,
+            }},
+        ];
+        Object.values(configLabels).forEach((label)=> {
+            fields.push({label: label + ' QC Status', show: true}
+            );
+        });
+
+        const tabs = [{id: 'browse', label: 'Browse'}];
+
+
+        return (
+            <Tabs tabs={tabs} defaultTab="browse" updateURL={true}>
+                <TabPane TabId={tabs[0].id}>
+                    <FilterableDataTable
+                        name="imaging_browser"
+                        data={this.state.data.Data}
+                        fields={fields}
+                        getFormattedCell={this.formatColumn}
+                    />
+                </TabPane>
+            </Tabs>
+        );
+    }
+}
+
+ImagingBrowserIndex.propTypes = {
+    dataURL: PropTypes.string.isRequired,
+    hasPermission: PropTypes.func.isRequired,
+};
+
+window.addEventListener('load', () => {
+    ReactDOM.render(
+        <ImagingBrowserIndex
+            dataURL={`${loris.BaseURL}/imaging_browser/?format=json`}
+            hasPermission={loris.userHasPermission}
```

Comment:
I don't believe you are currently using permission checks so you can take away lines `195` and `188`

Reply:
Removed!